operation.go

```diff
@@ -69,7 +69,7 @@ func NewOperationFromString(t, v, d string) (Operation, error) {
 	if err != nil {
 		return Operation{}, fmt.Errorf("Invalid value: \"%s\"", v)
 	}
-	date, err := time.Parse("2006-01-02", d)
+	date, err := time.Parse("02/01/2006", d)
```

Comment:
This change will be reflect format of the persistence data? I think that we need to maintain the standard format on the data storage in en-US format.